client/my-sites/plan-features-comparison/header.jsx

```diff
@@ -0,0 +1,445 @@
+import { getSiteSlug } from 'calypso/state/sites/selectors';
+import { planLevelsMatch } from 'calypso/lib/plans/index';
+
+export class PlanFeaturesComparisonHeader extends Component {
+	render() {
+		return this.renderPlansHeaderNoTabs();
+	}
+
+	renderPlansHeaderNoTabs() {
+		const {
+			newPlan,
+			bestValue,
+			planType,
+			popular,
+			selectedPlan,
+			isInSignup,
+			title,
+			translate,
+		} = this.props;
+
+		const headerClasses = classNames(
+			'plan-features-comparison__header',
+			getPlanClass( planType )
+		);
+
+		return (
+			<span>
+				<div>
+					{ popular && ! selectedPlan && (
+						<PlanPill isInSignup={ isInSignup }>{ translate( 'Popular' ) }</PlanPill>
+					) }
+				</div>
+				<header className={ headerClasses }>
+					<h4 className="plan-features-comparison__header-title">{ title }</h4>
+					{ planLevelsMatch( selectedPlan, planType ) && (
+						<PlanPill isInSignup={ isInSignup }>{ translate( 'Suggested' ) }</PlanPill>
+					) }
+					{ newPlan && ! selectedPlan && (
+						<PlanPill isInSignup={ isInSignup }>{ translate( 'New' ) }</PlanPill>
+					) }
+					{ bestValue && ! selectedPlan && (
+						<PlanPill isInSignup={ isInSignup }>{ translate( 'Best Value' ) }</PlanPill>
+					) }
+				</header>
+				<div className="plan-features-comparison__pricing">
+					{ this.getPlanFeaturesPrices() }
+					{ this.getBillingTimeframe() }
+					{ this.getAnnualDiscount() }
+					{ this.getIntervalDiscount() }
+				</div>
+			</span>
+		);
+	}
+
+	getDiscountTooltipMessage() {
+		const { currencyCode, currentSitePlan, translate, rawPrice, discountPrice } = this.props;
+		const price = formatCurrency( rawPrice, currencyCode );
+		const isDiscounted = !! discountPrice;
+
+		if ( planMatches( currentSitePlan.productSlug, { type: TYPE_FREE } ) ) {
+			return isDiscounted
+				? translate(
+						"You'll receive a discount for the first year. The plan will renew at %(price)s.",
+						{ args: { price } }
+				  )
+				: translate( 'Price for the next 12 months' );
+		}
+
+		return translate(
+			"You'll receive a discount from the full price of %(price)s because you already have a plan.",
+			{ args: { price } }
+		);
+	}
+
+	getPerMonthDescription() {
+		const {
+			discountPrice,
+			rawPrice,
+			rawPriceAnnual,
+			currencyCode,
+			translate,
+			planType,
+			currentSitePlan,
+			annualPricePerMonth,
+			isInSignup,
+			isMonthlyPlan,
+		} = this.props;
+
+		if ( isInSignup && isMonthlyPlan && annualPricePerMonth < rawPrice ) {
+			const discountRate = Math.round( ( 100 * ( rawPrice - annualPricePerMonth ) ) / rawPrice );
+			return translate( `Save %(discountRate)s%% by paying annually`, { args: { discountRate } } );
+		}
+
+		if ( isInSignup && ! isMonthlyPlan ) {
+			const annualPriceObj = getCurrencyObject( rawPriceAnnual, currencyCode );
+			const annualPriceText = `${ annualPriceObj.symbol }${ annualPriceObj.integer }`;
+
+			return translate( 'billed as %(price)s annually', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **13** times:
`translate( 'billed as %s annually' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/new-toggle-design)